lib/src/pages/conversation.dart

```diff
@@ -1,3 +1,5 @@
+import "package:btc_market/src/data/report.dart";
+import "package:btc_market/src/widgets/atomic/report_dialogue.dart";
```

Comment:
Instead of importing so deep, try importing just `package:btc_market/data.dart` or `/widgets.dart`. These correspond to files in `lib` that `export` all those deep files so we don't need to keep track of them one-by-one, and allow "mass importing" all our data or widgets conveniently. 

So basically, add the correct exports to those two files and you should be able to use the shorter versions